lib/clowder-common-ruby/schema.json

```diff
@@ -240,6 +245,10 @@
                     "description": "Defines the secret key for the specified bucket.",
                     "type": "string"
                 },
+                "region": {
```

Comment:
@hsong-rh is the region configurable by manageiq-loggers?

Reply:
@slemrmartin No, I don't think so. manageiq-loggers uses `ACCESS_KEY_ID` and `SECRET_KEY`